share/functions/fish_default_key_bindings.fish

```diff
@@ -13,7 +13,7 @@ function fish_default_key_bindings -d "Default (Emacs-like) key bindings for fis
     end
 
     # This is the default binding, i.e. the one used if no other binding matches
-    bind $argv "" self-insert
+    bind $argv \uE000 self-insert
```

Comment:
> I can't think of a way to get around using a private key binding as the bind character... vi normal mode...

I haven't given that situation any thought but the fact remains we should not introduce the use of Unicode private-use chars in our scripts. Not even if we give them symbolic names that are mapped to the range of chars that fish explicitly reserves for its own use. Those chars to really meant to be "internal use only" and thus not visible or usable in scripts or user input.

It seems like it should be possible to define a Unicode private-use char that fish uses when it sees `\c@` or equivalent in order to avoid the C-string null character problem. If that isn't possible without resorting to a major rewrite to avoid the use of `wchar_t *` objects then we have to live with the inability to `bind \c@`.